src/bundle/Resources/public/js/scripts/admin.url.wildcards.create.js

```diff
@@ -0,0 +1,32 @@
+(function (global, doc) {
+    const modal = doc.querySelector('#create-wildcards-modal');
+
+    if (modal) {
+        const discardBtns = modal.querySelectorAll('[data-dismiss="modal"]');
+        const submitBtn = modal.querySelector('[type="submit"]');
+        const inputs = [...modal.querySelectorAll('[required="required"]')];
+        const checkboxes = modal.querySelectorAll('.ez-field-edit--ezboolean input');
+        const toggleButtonState = () => {
+            const isInvalid = inputs.reduce((result, input) => result || input.value.trim().length === 0, false);
+            const methodName = isInvalid ? 'setAttribute' : 'removeAttribute';
+
+            submitBtn[methodName]('disabled', true);
+        };
+        const toggleCheckbox = (event) => {
+            const checkbox = event.target;
+            const methodName = checkbox.checked ? 'add' : 'remove';
+
+            checkbox.closest('.ez-data-source__label').classList[methodName]('is-checked');
```

Comment:
```suggestion
            checkbox.closest('.ez-data-source__label').classList.toggle('is-checked', checkbox.checked);
```

It is also an option here but this is a nitpick.